src/go/rpk/pkg/cli/group/group.go

```diff
@@ -135,6 +149,20 @@ The STATE columns shows which state the group is in:
 			groups := listed.Sorted()
 			isV4Response := slices.ContainsFunc(groups, func(g kadm.ListedGroup) bool { return g.State != "" })
 
+			response := groupListResponses{}
```

Comment:
Can we use this built `response` type to print the response in text format below? this way we don't have 2 different sources.